### PR TITLE
Style Dashboard Section

### PR DESCRIPTION
**Pull request by Pawko23**
### Changes:
* Applied styles for dashboard section
* Applied responsive styles for breakpoints: `768px`, `992px`, `1200px`

### Preview:
dashboard

### CODE REVIEW

File: scss/components/_dashboard.scss [hunk 1]
```diff
@@ -0,0 +1,103 @@
+@use '../abstracts' as *;
+
+.dashboard {
+    @include flexbox(column, center, center);
+    margin: 1.8rem 0;
+    &__info {
+        align-self: flex-start;
+        width: 80vw;
```
Comment (zniszcz) [inline scss/components/_dashboard.scss]: why vw not %? You don't need to use vw

File: scss/components/_dashboard.scss [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    }
+
+    &__description {
+        font-size: 0.9rem;
+    }
+
+    &__card {
+        @include flexbox (column, center, center);
+        width: 100%;
+        max-width: 34rem;
```
Comment (zniszcz) [inline scss/components/_dashboard.scss:27]: :+1:

File: scss/components/_dashboard.scss [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        &__card {
+            max-width: 43rem;
+        }
+    }
+}
+
+@media(min-width: 992px) {
+    .dashboard {
+        &__info {
+            width: 50vw;
```
Comment (zniszcz) [inline scss/components/_dashboard.scss]: same - don't use that units if you don't have to

**Review by zniszcz — CHANGES_REQUESTED** [review]

File: scss/components/_dashboard.scss (hunk 1 above)
Comment (Pawko23) [reply scss/components/_dashboard.scss]: True, I could just stay with %. I'll change that

**Review by zniszcz — APPROVED** [review]

**Pull request merged**